src/main/java/com/stuypulse/robot/util/ArmVisualizer.java

```diff
@@ -80,6 +85,8 @@ public ArmVisualizer() {
         baseLigament.setAngle(-90);
         
         SmartDashboard.putData("Arm Mech2d", arm);
+
+        fieldObject =  IOdometry.getInstance().getField().getObject("Field Arm");
```

Comment:
this is a bad spot to implement this as this class is just supposed to represent the mech2d window for the arms. i would recommend in the swerve drive maybe? where all the poses are initialized. or maybe move everything to odometry idk